definitions/variable/technology/electricity-operation.yaml

```diff
@@ -91,3 +92,19 @@
     description: Marginal Cost associated to the Frequency Containment requirement
     unit: EUR_2020/MWh
     skip-region-aggregation: true
+    
+- Export|Electricity:
+    description: Exported electricity from a given region
+    unit: MWh
+- Import|Electricity:
+    description: Exported electricity from a given region
+    unit: MWh
+- Non served Demand|Electricity:
```

Comment:
I'd rather call this variable "Unserved Demand" or "Curtailed Demand"

Reply:
I suggest the name `Not-Served Demand|Electricity`. I also agree with the "Unserved Demand".

Reply:
@danielhuppmann Curtailed demand does not fit as it is sth different: it refers in general to voluntary energy curtailments, while non served is missing energy
I will change to Unserved
@erikfilias I don't much like the '-' in the variable name